.github/workflows/main.yml

```diff
@@ -32,23 +32,15 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Install ruff
-        run: |
-          python3 -m pip install --break-system-packages ruff mypy
+      - name: Install tox
+        run: python -m pip install tox
       - name: Run ruff format
         run: |
-          ruff --version
-          if ! ruff format --check --quiet varlink
-          then
-              echo "Please run 'ruff format' on the above files or apply the diffs below manually"
-              ruff format --check --quiet --diff varlink
-          fi
+          tox -e ruff-format
       - name: Run ruff check
         run: |
-          ruff --version
-          ruff check varlink
+          tox -e ruff-check
       - name: Type Checking (mypy)
         run: |
-          python3 -m mypy --version
-          python3 -m mypy varlink/
+          tox -e py3-mypy
```

Comment:
I'm really not a tox expert, but sine neither of the ruff tox envs have the `py3-` prefix, is it necessary here?

Reply:
No, it's not needed especially if you don't need tox chooing a python version for the env.  I can remove that.